clippy_lints/src/identity_op.rs

```diff
@@ -58,15 +58,28 @@ impl<'a, 'tcx> LateLintPass<'a, 'tcx> for IdentityOp {
     }
 }
 
+fn no_zeros(v: &ConstInt) -> bool {
+    match *v {
+        ConstInt::I8(i) => i.count_zeros() == 0,
```

Comment:
can we just compare with `!0` or something?

Reply:
Indeed, simplified